techniques/silverlight/SL10.html

```diff
@@ -95,7 +95,7 @@ <h3>Two form fields with Submit</h3>
     						A full implementation might replace this with a server side data
     						object infrastructure. A full implementation might also provide a "Reset" or "Edit" button
     						to enable form submission again if there were issues. </p>
-            <pre xml:space="preserve">private void SubmitButton_Click(object sender, RoutedEventArgs e)
+            <pre>private void SubmitButton_Click(object sender, RoutedEventArgs e)
```

Comment:
```suggestion
            <pre><code>private void SubmitButton_Click(object sender, RoutedEventArgs e)
```
and add matching `</code>`